ci-operator/step-registry/ibmcloud/provision/vpc/endpoints-gateway/ibmcloud-provision-vpc-endpoints-gateway-commands.sh

```diff
@@ -105,11 +158,19 @@ vpcID=$(cat "${vpc_info_file}" | jq -r '.vpc.id')
 sgID=$(cat "${vpc_info_file}" | jq -r '.vpc.default_security_group.id')
 subnetID=$(cat "${vpc_info_file}" | jq -r '.subnets[0].id')
 clusterName="${NAMESPACE}-${UNIQUE_HASH}"
-allTargetsFile=$(mktemp)
+allTargetsFile="${ARTIFACT_DIR}/ep_targets.json"
 ibmcloud is endpoint-gateway-targets -q -output JSON > ${allTargetsFile} || exit 1
 run_command "ibmcloud is security-group-rule-add ${sgID} inbound tcp --remote '0.0.0.0/0' --port-min=443 --port-max=443" || exit 1
 
-srv_array=("IAM" "VPC" "ResourceController" "ResourceManager" "DNSServices" "COS" "GlobalSearch" "GlobalTagging")
+isPre418="True"
+isPreVersion "4.18" || isPre418="False"
+echo "is Pre 4.18 version: ${isPre418}"
```

Comment:
Similar to https://github.com/openshift/release/pull/58984#discussion_r1861447054 , now we can control `SERVICE_ENDPOINT_*` on the job config level for different version of OCP, we do not need the 4.18 version check any more.

```
srv_array=("IAM" "VPC" "ResourceController" "ResourceManager" "DNSServices" "COS" "GlobalSearch" "GlobalTagging" "KeyProtect" "COSConfig" "GlobalCatalog")
```